apps/dashboard/app/models/user_configuration.rb

```diff
@@ -77,7 +94,7 @@ def pinned_apps_group_by
 
   def public_url
     path = ENV['OOD_PUBLIC_URL'] || fetch(:public_url, '/public')
-    Pathname.new path
+    path.start_with?('/') ? Pathname.new(path) : Pathname.new('/public')
```

Comment:
If we have the check here, do we need it in `custom_css_paths` too? If you remove it from `custom_css_paths` can you add the comment back here?

Reply:
No, we do not need the check in `custom_css_paths`, removing....
Will move the comment here.

Reply:
👍